tools/ARIAtools/extractProduct.py

```diff
@@ -179,12 +178,12 @@ def prep_mask(product_dict, maskfilename, bbox_file, prods_TOTbbox, proj, amp_th
     # Download mask
     if maskfilename.lower()=='download':
         maskfilename=os.path.join(workdir,'watermask'+'.msk')
-
+        os.environ['CPL_ZIP_ENCODING'] = 'UTF-8'
         ###Make coastlines/islands union VRT
-        os.system('ogrmerge.py -o ' + os.path.join(workdir,'watermsk_shorelines.vrt') + ''.join(_world_watermask[::2]) + ' -field_strategy Union -f VRT -single')
+        subprocess.call('ogrmerge.py -o ' + os.path.join(workdir,'watermsk_shorelines.vrt') + ''.join(_world_watermask[::2]) + ' -field_strategy Union -f VRT -single',shell=True)
 
         ###Make lakes/ponds union VRT
-        os.system('ogrmerge.py -o ' + os.path.join(workdir,'watermsk_lakes.vrt') + ''.join(_world_watermask[1::2]) + ' -field_strategy Union -f VRT -single')
+        subprocess.call('ogrmerge.py -o ' + os.path.join(workdir,'watermsk_lakes.vrt') + ''.join(_world_watermask[1::2]) + ' -field_strategy Union -f VRT -single',shell=True)
```

Comment:
@ehavazli is there a particular reason why we should system from an "os" call to "subprocess"?

Reply:
review packages always flag os.system() as an issue. while the same is correct for subprocess, it is raising less flags. As I mentioned in the main comment we should also move away from subprocess as well.